lethal-squirrels/workforce-management
# Chaz training detail

## REFERENCE

Ticket: (#10 )

## DESCRIPTION
In `Models/EmployeeTraining.cs` - changed `TrainingID` to `TrainingProgramID` - this cleared up an error in the DB that was causing training programs and employees in training programs not to show up. 

In` Data.DbInitializer.cs` - Updated all the EmployeeTraining info from  `TrainingID` to `TrainingProgramID` as well. 

in `Views/TrainingPrograms/Details.cshtml` added the table for Employees and populated with a list of employees in that training program

## STEPS TO TEST
1. Clone this branch to a local repo
1. Delete current database by going to the `SQL Server Object Explorer`, navigating to the database tables and deleting all the tables (anything that starts with `dbo.`)
1. In the package manager, run `update-database`. 
1. Test functionality
	* Navigate to the `Training Programs` Tab and click on details view for a training program
	* make sure list of employees are being shown
